packages/mdc-tab/package.json

```diff
@@ -9,7 +9,6 @@
     "material design",
     "tab"
   ],
-  "main": "index.js",
```

Comment:
removed this because this isn't a package. Developer should be installing mdc-tabs

Reply:
This is incorrect. mdc-tab is part of the new tabs work being done in the feat/tabs/tabs branch. Users will be installing this as well as mdc-tab-bar, mdc-tab-scroller, and mdc-tab-indicator.

Reply:
Ah gotcha...I saw the `export {MDCTabFoundation, MDCTab} from './tab';` line in mdc-tabs, and thought it to be this mdc-tab package. Just missing a `.` to become `../tab`

Reply:
This was never updated after my comment. Also, mdc-tab-bar, mdc-tab-scroller, and mdc-tab-indicator were added since this.